### ISSUE TITLE
Please bump go to 1.12.5

### ISSUE THREAD

**Issue by georgethebeatle** [cloudfoundry/windows-tools-release#18]

**Comment by kieron-dev** [comment]
Hello! We see the story is blocked on bosh-golang-release updating to 1.12.5. But the [latest 0.27 release](https://github.com/bosh-packages/golang-release/archive/v0.27.0.tar.gz) from May 7 has 1.12.5 in the tarball 🙂

**Comment by arjun024** [comment]
@kieron-pivotal Hi Kieron, we see that v0.27 is not released yet. The latest release is v0.20 which only has go1.12.1. Do you know if v0.27 will be released soon?

**Comment by kieron-dev** [comment]
Maybe check with the BOSH team, but I think they only fill in release notes
when they get around to it every few months. E.g. see final comment in this
issue https://github.com/bosh-packages/golang-release/issues/11 where
release notes for 10 previous tags were added retrospectively.

In my previous team (services enablement), we treated any tag, with release
notes or not, as a viable candidate, especially if we had a security team
story added to bump the go version.
